ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/CentOS.yml

```diff
@@ -167,3 +167,24 @@
   when:
     - ansible_architecture == "x86_64"
   tags: expat
+
+
+############################
+# Test tools               #
+############################
+- name: Create Japanese locale
+  shell: localedef -i ja_JP -c -f UTF-8 ja_JP.UTF-8
+  tags: test_tools
```

Comment:
Would this likely be the same on RHEL?

Reply:
RHEL set up these locales with default installation, but I added the same code with condition check.